src/modules/upcoming-events.module/fields.json

```diff
@@ -1,4 +1,27 @@
 [
+  {
+    "label": "Set a form",
+    "name": "config",
```

Comment:
Why do we need this for upcoming events?

Reply:
We definitely don't need it, it's just that the upcoming-events module/app was breaking without it due to the way they're sharing components right now. Both apps share a method of fetching submission data and filtering it based on the form id in `event.functions/membership.js`. 

While bootstrapping this together, I included in both just to get things working again, but I'm planning on removing it.

Reply:
Removed in https://github.com/HubSpot/cms-event-registration/pull/49/commits/5aeac7229b301ad96aae838c45fbf5ebd1f0dd12